omgeo/services/__init__.py

```diff
@@ -1,7 +1,7 @@
-from .bing import Bing
-from .esri import EsriWGS
-from .us_census import USCensus
-from .mapquest import MapQuest, MapQuestSSL
-from .nominatim import Nominatim
-from .pelias import Pelias
-from .google import Google
+from .bing import Bing  # NOQA
```

Comment:
Is there a single warning type specific to the imports that can be `NOQA`'d for the file instead?